docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -323,4 +324,26 @@ realm-name:realm-two
 
 There are other options that can be specified for converting a filesystem realm, the details of which can be found using the `./bin/elytron-tool.sh filesystem-realm-integrity -h` command. 
 
+== Updating WildFly Elytron Tool Generated CLI Scripts for Domain Mode
+As mentioned above, some of the WildFly Elytron Tool commands create a CLI script that can be used to add resources to
+the Elytron subsystem. Some examples of these CLI scripts include:
+
+. The <FILESYSTEM-REALM-NAME>.cli script generated when the `filesystem-realm-encrypt` function with the WildFly Elytron Tool is used to encrypt an existing filesystem realm.
```

Comment:
Just minor, but I think it would be good to reword "function with the WildFly Elytron Tool" to just "command" here and below.

Reply:
fixed! Thank you @fjuma